.github/workflows/checks.yml

```diff
@@ -314,8 +314,13 @@ jobs:
         run: |
           poetry run poe circular_imports_check
 
+      - uses: asdf-vm/actions/setup@v3
+
+#     TODO(#1498): Remove contracts compilation in the step below once issue with compilation is resolved
       - name: Run tests
         run: |
+          poetry run poe compile_contracts v2
+          poetry run poe compile_contracts v1
```

Comment:
For some reason, compiled contract can't be found while running tests. Here is example CI run which this problem:
https://github.com/software-mansion/starknet.py/actions/runs/13627219872/job/38088663029

This is a temporary adjustment, which will be addresses in incoming PR.